perfkitbenchmarker/linux_benchmarks/hadoop_terasort_benchmark.py

```diff
@@ -54,7 +58,7 @@
 def GetConfig(user_config):
   config = configs.LoadConfig(BENCHMARK_CONFIG, user_config, BENCHMARK_NAME)
   if FLAGS['num_vms'].present:
-    config['vm_groups']['default']['vm_count'] = FLAGS.num_vms
+    config['vm_groups']['workers']['vm_count'] = FLAGS.num_vms - 1
```

Comment:
Based on the description I would expect this to just be FLAGS.num_vms not FLAGS.num_vms -1.